Makefile

```diff
@@ -149,8 +153,9 @@ local-unit-test: ## Run backend tests in the dev environment
 	fi
 
 .PHONY: local-functional-test
+local-functional-test: export AWS_PROFILE=$(TEST_AWS_PROFILE)
 local-functional-test: ## Run functional tests in the dev environment
-	docker-compose exec -T backend bash -c "cd /corpora-data-portal && export DEPLOYMENT_STAGE=test && make container-functionaltest"
+	chamber -b secretsmanager exec corpora/backend/$${DEPLOYMENT_STAGE}/auth0-secret -- docker-compose exec -T -e CLIENT_ID -e CLIENT_SECRET -e TEST_ACCOUNT_PASSWORD -e DEPLOYMENT_STAGE backend bash -c "cd /corpora-data-portal && make container-functionaltest"
```

Comment:
what does `--` do?

Reply:
Tells chamber that everything past the `--` is the command to run in the shell it creates, with the environment variables set from the values in the secrets manager